shell.nix

```diff
@@ -1,12 +1,23 @@
 { localLib ? import ./lib.nix
 , pkgs ? import (localLib.fetchNixPkgs) {}
+, io-dev-mode ? false
 }:
 
 let
   extraDeps = with pkgs;
-    [ wget awscli nodejs # for scripts/aws.hs
+    [ wget awscli  # for scripts/aws.hs
+      nodejs       # for cardano-sl/scripts/js/genesis-hash.js
+                   #     ..which also needs `npm install blakejs canonical-json`
     ];
-  drv = pkgs.haskell.lib.overrideCabal
-         (import ./default.nix {}).iohk-ops
-         (drv: { libraryHaskellDepends = extraDeps; });
-in drv.env
+  iohk-ops = pkgs.haskell.lib.overrideCabal
+             (import ./default.nix {}).iohk-ops
+             (drv: { libraryHaskellDepends = extraDeps; });
+  ioAlias = if io-dev-mode
+            then ''runhaskell -iiohk iohk/iohk-ops.hs''
```

Comment:
So we can now remove `iohk/iohk-env.nix`?

Reply:
Tough question -- I hoped we could retain the ability to use `io` outside `nix-shell`.

But if you deem it necessary, sure.